<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiPlayerDialog</class>
 <widget class="QDialog" name="MidiPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI Player</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="6" colspan="5">
      <widget class="QListWidget" name="playList"/>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>00:00 / 00:00</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="fastForwardButton">
       <property name="text">
        <string>Fast forward</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4" colspan="2">
      <widget class="QPushButton" name="tempoButton">
       <property name="text">
        <string>Set tempo</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="5">
      <widget class="QSlider" name="positionSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="moveUpButton">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="moveDownButton">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="addListButton">
       <property name="text">
        <string>Add List</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>addListButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>moveUpButton</tabstop>
  <tabstop>moveDownButton</tabstop>
  <tabstop>positionSlider</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>fastForwardButton</tabstop>
  <tabstop>tempoButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
